README.md

```diff
@@ -1,36 +1,30 @@
 Rails Sample App on OpenShift
 ============================
 
-This is a quickstart Rails application for OpenShift v3.
+This is a quickstart Rails application for OpenShift v3 that you can use as the starting point to develop your own and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
```

Comment:
Suggestion - `... that you can use as a starting point to develop your own application and deploy it in OpenShift cluster`
... not a native speaker, just sounds better in my head :P